src/components/bank-account/ba-card.tsx

```diff
@@ -0,0 +1,41 @@
+"use client";
+
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from "@/components/ui/card";
+import { BankAccount } from "@prisma/client";
+import Image from "next/image";
+import { DeleteBankAccountButton } from "./delete-ba-button";
+
+interface Props {
+  bankAccount: BankAccount;
+  session: { token: string; userId: string; name: string };
+  onDelete?: () => void;
+}
+
+export default function BankAccountCard({ bankAccount, session, onDelete }: Props) {
```

Comment:
Now Card is not a link - and now you can't send transactions, because you can't access the bank account detail :)) Can you please wrap this into `Link` component?